### ISSUE TITLE
radical-pilot-stats broken in recent RCT releases

### ISSUE THREAD

**Issue by eirrgang** [radical-cybertools/radical.pilot#2401]
It seems that the radical-pilot-stats script was not updated for a change in the states enumeration at some point.

```shell
(py37) login3.frontera(1010)$ radical-stack

  python               : /home1/07305/rpilot/eirrgang/py37/bin/python3
  pythonpath           : /opt/apps/intel19/impi19_0/python3/3.7.0/lib/python3.7/site-packages
  version              : 3.7.0
  virtualenv           : /home1/07305/rpilot/eirrgang/py37

  radical.gtod         : 1.5.0
  radical.pilot        : 1.6.6
  radical.saga         : 1.6.6
  radical.utils        : 1.6.6

(py37) login3.frontera(1019)$ RADICAL_VERBOSE=DEBUG RADICAL_PILOT_VERBOSE=DEBUG radical-pilot-stats --help
Traceback (most recent call last):
  File "/home1/07305/rpilot/eirrgang/py37/bin/radical-pilot-stats", line 36, in <module>
    rp.PENDING_LAUNCH          :  1,
AttributeError: module 'radical.pilot' has no attribute 'PENDING_LAUNCH'
```

**Comment by andre-merzky** [comment]
oh boy, that script is so badly out of sync :-/  Thanks for noting it!

**Comment by AymenFJA** [comment]
This is fixed with this PR: #2496

**Comment by mtitov** [comment]
resolved (#2496, #2605)
